getnotes/getnotes.py

```diff
@@ -414,8 +422,65 @@ async def findplayer(self, ctx, *,player: Union[ipaddress.IPv4Address, int, str]
         
         except ModuleNotFoundError:
             return await message.edit(content="`mysql-connector` requirement not found! Please install this requirement using `pip install mysql-connector`.")
-                      
+
+
+    async def dph(self, ctx) -> dict: # Deaths per hour
```

Comment:
A lot is wrong with this.

1. Declaring a return type without returning anything, or even being used to return anything
2. No arg for the ckey, so this wont even work
3. This isn't defined as a command, so there's no way to actually trigger this via discord
4. Redundant comment

```suggestion
    @commands.command()
    async def dph(self, ctx, player:str):
```